nemo/collections/asr/modules/rnn_encoder.py

```diff
@@ -108,9 +108,12 @@ def __init__(
         if subsampling_conv_channels == -1:
             subsampling_conv_channels = proj_size
         if subsampling and subsampling_factor > 1:
-            if subsampling == 'stacking':
+            if subsampling in ['stacking', 'stacking_norm']:
```

Comment:
We need doc updates for this new stacking_norm argument, both here and conformer_encoder.py

Reply:
Added a note here for now. We have the docs in the StackingSubsampling for this one, but we don't have a section on these downsampling approaches in the docs.